.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: c
+sudo: required
+compiler:
+  - gcc
+os: linux
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install -y dpkg-dev debhelper dkms lintian
+- sudo apt-get install linux-headers-`uname -r`
+script:
+- make
+# Getting PACKAGE_VERSION release
+- PACKAGE_VERSION=$(cat dkms.conf | grep  -i  PACKAGE_VERSION= | cut -d '"' -f 2)
```

Comment:
Since dkms is written in shell and simply sources that file, we can use shell
to get the PACKAGE_VERSION, which would be shorter and more reliable:

    PACKAGE_VERSION=`. dkms.conf; echo $PACKAGE_VERSION`